packages/hadron-build/lib/target.js

```diff
@@ -699,7 +699,7 @@ class Target {
         version: debianVersion,
         bin: this.productName,
         section: debianSection,
-        depends: ['libsecret-1-0', 'gnome-keyring', 'libgconf-2-4'],
+        depends: ['libsecret-1-0', 'gnome-keyring'],
```

Comment:
I'm not actually sure about libsecret. Doesn't seem to be a dependency of electron, so presumably something we use needs it?

Reply:
Keytar does, yeah